src/main/java/it/cnr/isti/hpc/wikipedia/reader/WikipediaArticleReader.java

```diff
@@ -154,22 +156,26 @@ public void process(WikiArticle page, Siteinfo si) {
 				return;
 			}
 
-			if(page.isHelp()){
+			Boolean isHelp = integerNamespace.equals(NamespaceCode.HELP_NAMESPACE_KEY.code);
```

Comment:
Boolean with a small `b`; Java [Object.equals](https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#equals-java.lang.Object-) only returns true or false, never null.

Reply:
Actually, might be neater to just do away with the whole thing and do switch-case:
```
switch(integerNamespace) {
    case NamespaceCode.HELP_NAMESPACE_KEY.code:
        // stuff
        break;
    ...
}
```

Reply:
Good spot @hmcc

Reply:
switch case did look nicer but apparently I can't cause they're not constants :(